doc/integrations/duckdb.md

```diff
@@ -4,16 +4,11 @@ jupytext:
     extension: .md
     format_name: myst
     format_version: 0.13
-    jupytext_version: 1.14.4
+    jupytext_version: 1.14.5
 kernelspec:
   display_name: Python 3 (ipykernel)
   language: python
   name: python3
-myst:
```

Comment:
Why deleted this?

Reply:
Some merge error with master, reverted this file